pom.xml

```diff
@@ -21,21 +21,15 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <biodata.version>0.4.6</biodata.version>
+        <biodata.version>0.4.7</biodata.version>
```

Comment:
The Travis build is complaining about not finding the artifacts in "central" but they seem fine in https://oss.jfrog.org/artifactory/libs-release/uk/ac/ebi/eva/biodata-formats/ ... :thinking: